src/Bindings/ManualBindings_World.cpp

```diff
@@ -830,6 +875,39 @@ static int tolua_cWorld_ScheduleTask(lua_State * tolua_S)
 
 
 
+static int tolua_cWorld_SpawnSplitExperienceOrbs(lua_State* tolua_S)
+{
+	cLuaState L(tolua_S);
+	if (
+		!L.CheckParamSelf("cWorld") ||
+		!L.CheckParamNumber(2, 5) ||
+		!L.CheckParamEnd(6)
+	)
+	{
+		return 0;
+	}
+
+	cWorld * self = nullptr;
+	double X = 0;
+	double Y = 0;
+	double Z = 0;
+	int Reward = 0;
+	L.GetStackValues(1, self, X, Y, Z, Reward);
```

Comment:
I don't think we are on same page.
this is another function `SpawnExperienceOrb`, do I need to change this one to vector3d ??
Do I need to change this function parameter list?

Reply:
I mis-understand your request, I ported DoWithNearestPlayer with vector3d interface

Reply:
Sorry for the confusion, `SpawnExperienceOrb` is already exported to the API so I don't expect you to change it.  It's also good that `DoWithNearestPlayer` is exported with vectors too.

We're very slowly trying to change functions with three separate numbers as a coordinate to using `Vector3`s and all new lua exports should be using the new style.  It's harder to deal with changes to the lua API because the deprecated interfaces have to be maintained via manual bindings and plug-in authors would have to change their code.